src/views/Provisioners/index.jsx

```diff
@@ -0,0 +1,49 @@
+import { hot } from 'react-hot-loader';
+import { Component } from 'react';
+import { Switch } from 'react-router-dom';
+import RouteWithProps from '../../components/RouteWithProps';
+import loadable from '../../utils/loadable';
+
+const ViewProvisioners = loadable(() =>
+  import(/* webpackChunkName: 'Provisioners.ViewProvisioners' */ './ViewProvisioners')
+);
+const ViewWorkerTypes = loadable(() =>
+  import(/* webpackChunkName: 'Provisioners.ViewWorkerTypes' */ './ViewWorkerTypes')
+);
+const ViewWorker = loadable(() =>
+  import(/* webpackChunkName: 'Provisioners.ViewWorker' */ './ViewWorker')
+);
+const ViewWorkers = loadable(() =>
+  import(/* webpackChunkName: 'Provisioners.ViewWorkers' */ './ViewWorkers')
+);
+
+@hot(module)
+export default class Provisioners extends Component {
+  render() {
+    const {
+      match: { path },
+      ...props
+    } = this.props;
+
+    return (
+      <Switch>
+        <RouteWithProps
+          path={`${path}/:provisionerId/worker-types/:workerType/workers/:workerGroup?/:workerId?`}
```

Comment:
I see `workerGroup` and `workerId` are optional, but I couldn't find how their presence affected a view. Could you point me to the right place?

Reply:
This needs to be removed. We don't have dropdowns in the worker view anymore since most users come to this page via the workers view. I will revise.